samples/metrics_monitor/test/cttmetrics_gtest.cpp

```diff
@@ -123,20 +121,16 @@ int setGpuFrequency(int min, int max, int boost)
     {
         if (min < getGpuFrequency(GPU_MIN_FREQ_FILE_PATH))
```

Comment:
what is the purpose of this "if" if you do the same things in both branches? the order of writing matters?

Reply:
Yes, the order depends on the current min frequency.

Reply:
But what is the effect of different order of writing from HW side?

Reply:
For instance if we have: gt_min_freq_mhz=700 / gt_max_freq_mhz=700 / gt_boost_freq_mhz=700 and we want to set frequency to 1000, we need write to files in this order: gt_boost_freq_mhz, gt_max_freq_mhz, gt_min_freq_mhz.
Overwise we wil get: 
# echo 1000 > /sys/class/drm/card0/gt_min_freq_mhz
-bash: echo: write error: Invalid argument
and -1 must be returned from write(...)

Reply:
OK, now I got the idea